config

```diff
@@ -1,3 +1,12 @@
 ngx_addon_name=ngx_http_lua_ssl_module
-HTTP_MODULES="$HTTP_MODULES ngx_http_lua_ssl_module"
-NGX_ADDON_SRCS="$NGX_ADDON_SRCS $ngx_addon_dir/src/ngx_http_lua_ssl_module.c"
+
+if test -n "$ngx_module_link"; then
+  ngx_module_type=HTTP
+  ngx_module_name=ngx_http_lua_ssl_module
+  ngx_module_srcs="$ngx_addon_dir/src/ngx_http_lua_ssl_module.c"
```

Comment:
Can you put the common part (the module name, the module files) into a variable to avoid repeating between two kinds of configure?

Reply:
Sure! I'm changing here...